.github/workflows/static-analysis.yml

```diff
@@ -30,6 +30,7 @@ jobs:
             **/node_modules
             .yarn
           key: ${{ runner.os }}-yarn-cache-${{ hashFiles('./yarn.lock') }}
+          restore-keys: ${{ runner.os }}-yarn-cache-
```

Comment:
flagging this key is still here although you said it was a leftover

Reply:
thanks - forgot to update these two other files